.github/workflows/build.yaml

```diff
@@ -104,25 +104,27 @@ jobs:
     runs-on: macos-latest
     steps:
       - name: Set build environment
-        run: echo "GOARCH=${{ matrix.arch }}" >> $GITHUB_ENV
+        run: |
+          echo "GOARCH=${{ matrix.arch }}" >> $GITHUB_ENV
+          echo "GO_BUILD_DIR=lp-builds/" >> $GITHUB_ENV
```

Comment:
could not find a way to generate a random build directory name and still use it later for codesigning stage. hardcoded as `lp-builds` here until then.